tests/sdk/v2/driver/test_ce_runtime.py

```diff
@@ -1143,14 +1144,17 @@ def test_ignore_task_inv_id(
 
         # Then
         mocked_client.get_workflow_run_logs.assert_called_once_with(workflow_run_id)
-        assert logs == {
-            "UNKNOWN TASK INV ID": [
-                "<message sentinel 1>",
-                "<message sentinel 2>",
-                "<message sentinel 3>",
-                "<message sentinel 4>",
-            ]
-        }
+        assert logs == WorkflowLogs(
+            {
+                "UNKNOWN TASK INV ID": [
+                    "<message sentinel 1>",
+                    "<message sentinel 2>",
+                    "<message sentinel 3>",
+                    "<message sentinel 4>",
+                ]
+            },
+            [],
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏
```suggestion
        assert logs == WorkflowLogs(
            per_task={
                "UNKNOWN TASK INV ID": [
                    "<message sentinel 1>",
                    "<message sentinel 2>",
                    "<message sentinel 3>",
                    "<message sentinel 4>",
                ]
            },
            env_setup=[],